server/routes/api.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express')
+const WeatherApi = require('./services')
+const weatherApi = new WeatherApi
+const router = express.Router()
+const moment = require('moment')
+const City = require('../models/city')
+
+const updateDbHourly = async function () {
+    const result = await City.find({}, { _id: 0, openWeatherId: 1 })
+    const citiesIds = []
+    result.forEach(id => citiesIds.push(id.openWeatherId))
+    const updatingResponse = await weatherApi.getCitiesById(citiesIds)
+    const updatedCities = updatingResponse.data.list.map(city => {
+        return {
+            name: city.name,
+            openWeatherId: city.id,
+            temprature: Math.floor(city.main.temp) + '°',
+            condition: city.weather[0].description,
+            conditionPic: `animated/${city.weather[0].icon}.svg`,
+            date: new Date()
+        }
+    })
+    updatedCities.forEach(city => {
+        City.findOneAndUpdate({ openWeatherId: city.openWeatherId },
+            {
+                temprature: city.temprature,
+                condition: city.condition,
+                conditionPic: city.conditionPic,
+                date: city.date
+            },
+            { new: true }, function (err, result) {
+                console.log("just updated the database");
+            })
+    })
+}
+const hour = 1000 * 60 * 60
```

Comment:
when you have a true constant like this in javascript you usually declare it at the top of the file in all caps
https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/